tests/console/journald_fss.pm

```diff
@@ -0,0 +1,33 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Case 1463314  - FIPS: systemd
+
+use base "consoletest";
+use strict;
+use testapi;
+
+sub run() {
+
+    select_console "root-console";
+    assert_script_run("echo 'Seal=yes' >> /etc/systemd/journald.conf");
+    wait_still_screen;
+    assert_script_run("mkdir -p /var/log/journal");
+    wait_still_screen;
+    assert_script_run("systemctl restart systemd-journald.service");
+    wait_still_screen;
+    assert_script_run("journalctl --setup-keys");
+    wait_still_screen;
+    assert_script_run("journalctl --setup-keys --force | tee /tmp/key");
+    wait_still_screen;
+    assert_script_run("journalctl --verify --verify-key=`cat /tmp/key`");
```

Comment:
You shouldn't need any assert_still_screen between the assert_script_run calls, do you?